packages/vsx-registry/src/browser/vsx-extension.tsx

```diff
@@ -0,0 +1,441 @@
+        }
+        return <div className='theia-vsx-extension'>
+            <div className='theia-vsx-extension-icon' style={iconStyle} />
+            <div className='theia-vsx-extension-content'>
+                <div className='title'>
+                    <div className='noWrapInfo'>
+                        <span className='name'>{displayName}</span> <span className='version'>{version}</span>
+                    </div>
+                    <div className='stat'>
+                        {downloadCount && <span className='download-count'><i className='fa fa-download' />{downloadCompactFormatter.format(downloadCount)}</span>}
+                        {averageRating && <span className='average-rating'><i className='fa fa-star' />{averageRating.toFixed(1)}</span>}
+                    </div>
+                </div>
+                <div className='noWrapInfo theia-vsx-extension-description'>{description}</div>
+                <div className='theia-vsx-extension-action-bar'>
+                    <span className='noWrapInfo theia-vsx-extension-publisher'>{publisher}</span>
+                    {this.renderAction()}
+                </div>
+            </div>
+        </div >;
+    }
+}
+
+export class VSXExtensionEditorComponent extends AbstractVSXExtensionComponent {
+    render(): React.ReactNode {
+        const { builtin, preview, id, iconUrl, publisher, displayName, description, averageRating, downloadCount, repository, license, readme } = this.props.extension;
+        const iconStyle: React.CSSProperties = {};
+        if (iconUrl) {
+            iconStyle.backgroundImage = `url('${iconUrl}')`;
+        }
+        return <React.Fragment>
+            <div className='header'>
+                <div className='icon-container' style={iconStyle} />
+                <div className='details'>
+                    <div className='title'>
+                        <span title='Extension name' className='name' onClick={this.openExtension}>{displayName}</span>
+                        <span title='Extension identifier' className='identifier'>{id}</span>
+                        {preview && <span className='preview'>Preview</span>}
+                        {builtin && <span className='builtin'>Built-in</span>}
+                    </div>
+                    <div className='subtitle'>
+                        <span title='Publisher name' className='publisher' onClick={this.searchPublisher}>{publisher}</span>
+                        {downloadCount && <span className='fa fa-download download-count' onClick={this.openExtension}>{downloadFormatter.format(downloadCount)}</span>}
+                        {averageRating && <span className='average-rating' onClick={this.openAverageRating}>{this.renderStars()}</span>}
+                        {repository && <span className='repository' onClick={this.openRepository}>Repository</span>}
+                        {license && <span className='license' onClick={this.openLicense}>License</span>}
```

Comment:
Display the license type (`license` property) instead of "License".

https://github.com/eclipse/openvsx/blob/74dd60eebc6855411b91f67ff809c15d9ad68a7a/webui/src/pages/extension-detail/extension-detail.tsx#L244-L254